src/frontend/src/icp-eth/components/tokens/ManageTokens.svelte

```diff
@@ -84,6 +88,23 @@
 		$pseudoNetworkChainFusion
 	]);
 
+	// To avoid strange behavior when the exchange data changes (for example, the tokens may shift
+	// since some of them are sorted by market cap), we store the exchange data in a variable during
+	// the life of the component.
+	let exchangesStaticData: ExchangesData | undefined;
+	onMount(() => {
+		exchangesStaticData = $exchanges;
+	});
+
+	let allTokensSorted: Token[] = [];
+	$: exchangesStaticData,
```

Comment:
I don't think you need `$: exchangesStaticData,` here.

```
$: allTokensSorted = etc.
```

Should do no given that `exchangesStaticData` is use as param of the assignation?

Reply:
since it is initialized as `undefined`, i was worried that it would not consider the change when it moves from `undefined` to `nonNullish`  on `onMount` event

but you were right, it works either way

removed it